### PR TITLE
streamline init sync

### PR DESCRIPTION
**Pull request by dnbrwstr**
Before
Screenshot 2024-07-22 at 5 42 02 PM

After (critical info pops in faster + total duration is ~3-4s less)
Screenshot 2024-07-22 at 5 40 56 PM

### CODE REVIEW

**Comment by jamesacklin** [comment]
I assume ~25MB of the other 30MB are group images/avatars?

**Comment by dnbrwstr** [comment]
> I assume ~25MB of the other 30MB are group images/avatars?

lol most of it is (22mb) is the unminimized bundle itself, which gets tracked in this context. And the rest is 90% json.....

**Review by arthyn — APPROVED** [review]
lgtm, just one question in there

File: packages/shared/src/store/sync.ts [hunk 1]
```diff
@@ -969,70 +935,84 @@ export const handleDiscontinuity = async () => {
...
+    syncVolumeSettings({ priority: SyncPriority.Low }).then(() =>
+      reporter.log(`finished syncing volume settings`)
+    ),
+    syncStorageSettings({ priority: SyncPriority.Low }).then(() =>
+      reporter.log(`finished initializing storage`)
+    ),
+    syncPushNotificationsSetting({ priority: SyncPriority.Low }).then(() =>
+      reporter.log(`finished syncing push notifications setting`)
+    ),
+    syncBlockedUsers({ priority: SyncPriority.Low }).then(() => {
```
[the remark this answers is not in the document]
Comment (dnbrwstr) [reply packages/shared/src/store/sync.ts:974]: Ah good catch -- yeah I'm pretty sure we discussed it being unnecessary yesterday.

**Review by latter-bolden — APPROVED** [review]

**Pull request merged**